toolkits/jira/arcade_jira/client.py

```diff
@@ -2,83 +2,51 @@
 import json
 import json.decoder
 from dataclasses import dataclass
-from typing import Any, cast
+from typing import cast
 
 import httpx
+from arcade_tdk import ToolContext
+from arcade_tdk.errors import ToolExecutionError
 
-import arcade_jira.cache as cache
 from arcade_jira.constants import JIRA_API_VERSION, JIRA_BASE_URL, JIRA_MAX_CONCURRENT_REQUESTS
```

Comment:
this concurrency wil have to be set by the engine I think

Reply:
This internal cache has removed now